packages/calling-component-bindings/src/utils/callUtils.ts

```diff
@@ -103,6 +103,41 @@ export const _updateUserDisplayNames = (participants: RemoteParticipantState[]):
   }
 };
 
+/**
+ * Get the display name of a participant for given userId
+ *
+ * @internal
+ *
+ */
+export const getRemoteParticipantDisplayName = (
```

Comment:
Other properties beyond display name in future may want all remote participants ended.
A different way to structure this PR, would be to create a new selector `getAllRemoteParticipants` that returns active and previous participants. Then where `getRemoteParticipants, getRemoteParticipantsEnded` is used as arguments into the reselect, you can just use getAllRemoteParticipants knowing it will retrieve both active and previous participants.
Just a suggestion, otherwise PR looks great! Nice find on this property

Reply:
Will be implementing this in separate PR.